scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -464,6 +464,14 @@ class Router(formatOps: FormatOps) {
           Split(NewlineT(isDouble = tok.hasBlankLine), 0)
         )
 
+      case tok @ FormatToken(_: T.RightParen, _: T.KwDef, _)
+          if leftOwner.is[Defn.ExtensionGroup] =>
+        // Force a newline with the body of a Defn.ExtensionGroup if the
+        // method doesn't fit on a single line.
+        Seq(
+          Split(Space, 0).withSingleLine(leftOwner.tokens.last),
+          Split(Newline, 1).withIndent(2, leftOwner.tokens.last, After)
```

Comment:
The 2 space indentation is hardcoded in several places so I hardcoded the `2` literal here as well. We may need to make this indentation configuration down the road with significant indentation syntax.

Reply:
i think significant is only important for parsing. for formatting, you can fix it to 2, no?